man/integritytab.xml

```diff
@@ -72,6 +72,16 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>no-journal</option></term>
+
+        <listitem><para>
+        Disable the journal. Corresponds to the "direct writes" mode documented in https://docs.kernel.org/admin-guide/device-mapper/dm-integrity.html.
```

Comment:
`<ulink url="https://...">the dm-integrity documentation</ulink>`

Reply:
Done